.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
I relooked at this and saw that I gave you bad information when I first reviewed this. This article explains why using a _.dockerignore_ file is recommended for both security of secrets as well as to optimize your build times: https://codefresh.io/docker-tutorial/not-ignore-dockerignore/.

My recommendation is to blacklist all files and then whitelist the file you wish to copy to your Docker image.